app/interactors/cangaroo/perform_jobs.rb

```diff
@@ -15,7 +16,20 @@ def data
     end
 
     def enqueue_jobs(type, payload)
-      initialize_jobs(type, payload).select(&:perform?).each(&:enqueue)
+      initialize_jobs(type, payload).each do |job|
+        if job.perform?
+          # TODO log simplified info about payload info is being queued
+          job.enqueue
+        else
+          # TODO persist translation for audit trail?
+
+          log.info 'skipping job for payload',
+            skipped_job: job.class.to_s,
+            payload: payload,
+            payload_type: type,
+            payload_state: job.payload_state
```

Comment:
Whoops! Forgot to remove this. This relies on the `Translation` class

https://github.com/nebulab/cangaroo/pull/19/files#diff-45c84b17b9d53ae7c13d03d296304e38R27